src/app/wallets/ln-send-payment.ts

```diff
@@ -115,37 +105,30 @@ export const lnInvoicePaymentSend = async ({
   walletId,
   userId,
   logger,
-}: LnInvoicePaymentSendArgs): Promise<PaymentSendStatus | ApplicationError> =>
-  asyncRunInSpan(
-    "app.lnInvoicePaymentSend",
-    {
-      [SemanticAttributes.CODE_FUNCTION]: "lnInvoicePaymentSend",
-      "payment.initiation_method": PaymentInitiationMethod.Lightning,
-      "payment.wallet_id": walletId,
-      "payment.request": paymentRequest,
```

Comment:
`payment.request` is removed but not added back anywhere... or is it part of the arguments?

Reply:
yes, `paymentRequest` is an attribute